// [[Reason: Sourcemap is more detailed]] ////

//// [tests/cases/compiler/declarationEmitTypeParameterNameInOuterScope.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,6 +1,6 @@
 
 //// [declarationEmitTypeParameterNameInOuterScope.d.ts.map]
-{"version":3,"file":"declarationEmitTypeParameterNameInOuterScope.d.ts","sourceRoot":"","sources":["declarationEmitTypeParameterNameInOuterScope.ts"],"names":[],"mappings":"AAAA,cAAM,CAAC;CAAI;AAEX,QAAA,IAAI,CAAC,SAAW,CAAC,KAAG,CAAM,CAAC;AAC3B,iBAAS,EAAE,CAAC,CAAC,EAAG,CAAC,EAAE,CAAC,GAAG,CAAC,CAAa;AAErC,QAAA,IAAI,EAAE,SAAW,CAAC,KAAG,YAAuB,CAAC;AAC7C,iBAAS,EAAE,CAAC,CAAC,EAAG,CAAC,EAAE,CAAC,GAAG,UAAU,CAAC,CAAC,CAAmB;AAGtD,UAAU,CAAC;CAAI;AAEf,QAAA,IAAI,CAAC,SAAW,CAAC,KAAG,CAAM,CAAC;AAC3B,iBAAS,EAAE,CAAC,CAAC,EAAG,CAAC,EAAE,CAAC,GAAG,CAAC,CAAa"}
+{"version":3,"file":"declarationEmitTypeParameterNameInOuterScope.d.ts","sourceRoot":"","sources":["declarationEmitTypeParameterNameInOuterScope.ts"],"names":[],"mappings":"AAAA,cAAM,CAAC;CAAI;AAEX,QAAA,IAAI,CAAC,GAAI,CAAC,EAAG,CAAC,EAAE,CAAC,KAAG,CAAM,CAAC;AAC3B,iBAAS,EAAE,CAAC,CAAC,EAAG,CAAC,EAAE,CAAC,GAAG,CAAC,CAAa;AAErC,QAAA,IAAI,EAAE,GAAI,CAAC,EAAG,CAAC,EAAE,CAAC,KAAG,UAAU,CAAC,CAAY,CAAC;AAC7C,iBAAS,EAAE,CAAC,CAAC,EAAG,CAAC,EAAE,CAAC,GAAG,UAAU,CAAC,CAAC,CAAmB;AAGtD,UAAU,CAAC;CAAI;AAEf,QAAA,IAAI,CAAC,GAAI,CAAC,EAAG,CAAC,EAAE,CAAC,KAAG,CAAM,CAAC;AAC3B,iBAAS,EAAE,CAAC,CAAC,EAAG,CAAC,EAAE,CAAC,GAAG,CAAC,CAAa"}
 
-//// https://sokra.github.io/source-map-visualization#base64,ZGVjbGFyZSBjbGFzcyBBIHsNCn0NCmRlY2xhcmUgdmFyIGE6IDxBPih4OiBBKSA9PiBBOw0KZGVjbGFyZSBmdW5jdGlvbiBhMjxBPih4OiBBKTogQTsNCmRlY2xhcmUgdmFyIGEzOiA8QT4oeDogQSkgPT4gZ2xvYmFsVGhpcy5BOw0KZGVjbGFyZSBmdW5jdGlvbiBhNDxBPih4OiBBKTogZ2xvYmFsVGhpcy5BOw0KaW50ZXJmYWNlIEIgew0KfQ0KZGVjbGFyZSB2YXIgYjogPEI+KHg6IEIpID0+IEI7DQpkZWNsYXJlIGZ1bmN0aW9uIGIyPEI+KHg6IEIpOiBCOw0KLy8jIHNvdXJjZU1hcHBpbmdVUkw9ZGVjbGFyYXRpb25FbWl0VHlwZVBhcmFtZXRlck5hbWVJbk91dGVyU2NvcGUuZC50cy5tYXA=,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiZGVjbGFyYXRpb25FbWl0VHlwZVBhcmFtZXRlck5hbWVJbk91dGVyU2NvcGUuZC50cyIsInNvdXJjZVJvb3QiOiIiLCJzb3VyY2VzIjpbImRlY2xhcmF0aW9uRW1pdFR5cGVQYXJhbWV0ZXJOYW1lSW5PdXRlclNjb3BlLnRzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUFBLGNBQU0sQ0FBQztDQUFJO0FBRVgsUUFBQSxJQUFJLENBQUMsU0FBVyxDQUFDLEtBQUcsQ0FBTSxDQUFDO0FBQzNCLGlCQUFTLEVBQUUsQ0FBQyxDQUFDLEVBQUcsQ0FBQyxFQUFFLENBQUMsR0FBRyxDQUFDLENBQWE7QUFFckMsUUFBQSxJQUFJLEVBQUUsU0FBVyxDQUFDLEtBQUcsWUFBdUIsQ0FBQztBQUM3QyxpQkFBUyxFQUFFLENBQUMsQ0FBQyxFQUFHLENBQUMsRUFBRSxDQUFDLEdBQUcsVUFBVSxDQUFDLENBQUMsQ0FBbUI7QUFHdEQsVUFBVSxDQUFDO0NBQUk7QUFFZixRQUFBLElBQUksQ0FBQyxTQUFXLENBQUMsS0FBRyxDQUFNLENBQUM7QUFDM0IsaUJBQVMsRUFBRSxDQUFDLENBQUMsRUFBRyxDQUFDLEVBQUUsQ0FBQyxHQUFHLENBQUMsQ0FBYSJ9,Y2xhc3MgQSB7IH0KCnZhciBhID0gPEEsPih4OiBBKTogQSA9PiB4OwpmdW5jdGlvbiBhMjxBLD4oeDogQSk6IEEgeyByZXR1cm4geCB9Cgp2YXIgYTMgPSA8QSw+KHg6IEEpOiBnbG9iYWxUaGlzLkEgPT4gbmV3IEEoKTsKZnVuY3Rpb24gYTQ8QSw+KHg6IEEpOiBnbG9iYWxUaGlzLkEgeyByZXR1cm4gbmV3IEEoKSB9CgoKaW50ZXJmYWNlIEIgeyB9Cgp2YXIgYiA9IDxCLD4oeDogQik6IEIgPT4geDsKZnVuY3Rpb24gYjI8Qiw+KHg6IEIpOiBCIHsgcmV0dXJuIHggfQo=
+//// https://sokra.github.io/source-map-visualization#base64,ZGVjbGFyZSBjbGFzcyBBIHsNCn0NCmRlY2xhcmUgdmFyIGE6IDxBPih4OiBBKSA9PiBBOw0KZGVjbGFyZSBmdW5jdGlvbiBhMjxBPih4OiBBKTogQTsNCmRlY2xhcmUgdmFyIGEzOiA8QT4oeDogQSkgPT4gZ2xvYmFsVGhpcy5BOw0KZGVjbGFyZSBmdW5jdGlvbiBhNDxBPih4OiBBKTogZ2xvYmFsVGhpcy5BOw0KaW50ZXJmYWNlIEIgew0KfQ0KZGVjbGFyZSB2YXIgYjogPEI+KHg6IEIpID0+IEI7DQpkZWNsYXJlIGZ1bmN0aW9uIGIyPEI+KHg6IEIpOiBCOw0KLy8jIHNvdXJjZU1hcHBpbmdVUkw9ZGVjbGFyYXRpb25FbWl0VHlwZVBhcmFtZXRlck5hbWVJbk91dGVyU2NvcGUuZC50cy5tYXA=,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiZGVjbGFyYXRpb25FbWl0VHlwZVBhcmFtZXRlck5hbWVJbk91dGVyU2NvcGUuZC50cyIsInNvdXJjZVJvb3QiOiIiLCJzb3VyY2VzIjpbImRlY2xhcmF0aW9uRW1pdFR5cGVQYXJhbWV0ZXJOYW1lSW5PdXRlclNjb3BlLnRzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUFBLGNBQU0sQ0FBQztDQUFJO0FBRVgsUUFBQSxJQUFJLENBQUMsR0FBSSxDQUFDLEVBQUcsQ0FBQyxFQUFFLENBQUMsS0FBRyxDQUFNLENBQUM7QUFDM0IsaUJBQVMsRUFBRSxDQUFDLENBQUMsRUFBRyxDQUFDLEVBQUUsQ0FBQyxHQUFHLENBQUMsQ0FBYTtBQUVyQyxRQUFBLElBQUksRUFBRSxHQUFJLENBQUMsRUFBRyxDQUFDLEVBQUUsQ0FBQyxLQUFHLFVBQVUsQ0FBQyxDQUFZLENBQUM7QUFDN0MsaUJBQVMsRUFBRSxDQUFDLENBQUMsRUFBRyxDQUFDLEVBQUUsQ0FBQyxHQUFHLFVBQVUsQ0FBQyxDQUFDLENBQW1CO0FBR3RELFVBQVUsQ0FBQztDQUFJO0FBRWYsUUFBQSxJQUFJLENBQUMsR0FBSSxDQUFDLEVBQUcsQ0FBQyxFQUFFLENBQUMsS0FBRyxDQUFNLENBQUM7QUFDM0IsaUJBQVMsRUFBRSxDQUFDLENBQUMsRUFBRyxDQUFDLEVBQUUsQ0FBQyxHQUFHLENBQUMsQ0FBYSJ9,Y2xhc3MgQSB7IH0KCnZhciBhID0gPEEsPih4OiBBKTogQSA9PiB4OwpmdW5jdGlvbiBhMjxBLD4oeDogQSk6IEEgeyByZXR1cm4geCB9Cgp2YXIgYTMgPSA8QSw+KHg6IEEpOiBnbG9iYWxUaGlzLkEgPT4gbmV3IEEoKTsKZnVuY3Rpb24gYTQ8QSw+KHg6IEEpOiBnbG9iYWxUaGlzLkEgeyByZXR1cm4gbmV3IEEoKSB9CgoKaW50ZXJmYWNlIEIgeyB9Cgp2YXIgYiA9IDxCLD4oeDogQik6IEIgPT4geDsKZnVuY3Rpb24gYjI8Qiw+KHg6IEIpOiBCIHsgcmV0dXJuIHggfQo=
 
